### PR TITLE
Botonic react: property to disable trigger button notifications

### PR DESCRIPTION
**Pull request by Iru89**
## Description
Add property to disable trigger button notifications
Create interface for WebchatArgs

## Context
Trigger button notifications are disabled by default but with this property you can set true to enable notifications. 

## To document / Usage example
Captura de Pantalla 2023-07-20 a las 13 29 46

## Testing

The pull request...

- has unit tests
- has integration tests
- doesn't need tests because... **[provide a description]**

### CODE REVIEW

File: packages/botonic-react/src/index-types.ts [hunk 1]
```diff
@@ -96,28 +100,28 @@ export interface CustomMessageType {
...
-//   onInit?: (app: WebchatApp, args: any) => void
-//   onMessage?: (app: WebchatApp, message: WebchatMessage) => void
-//   onOpen?: (app: WebchatApp, args: any) => void
-//   onConnectionChange?: (app: WebchatApp, isOnline: boolean) => void
-//   persistentMenu?: PersistentMenuTheme
-//   storage?: Storage | null
-//   storageKey?: any
-//   theme?: ThemeProps
-// }
+export interface WebchatArgs {
```
Comment (vanbasten17) [inline packages/botonic-react/src/index-types.ts:103]: ❤️

**Review by vanbasten17 — APPROVED** [review]
Nice job @Iru89 !!

File: packages/botonic-react/src/components/index-types.ts [hunk 2]
```diff
@@ -151,7 +151,9 @@ export interface ThemeProps extends StyleProp {
     enableArrows?: boolean
   }
   reply?: StyleProp & CustomProp
-  triggerButton?: ImageProp & StyleProp & CustomProp
+  triggerButton?: { notificationsEnabled?: boolean } & ImageProp &
```
Comment (asastre) [inline packages/botonic-react/src/components/index-types.ts:154]: Suggestion: Why not adding a `TriggerButtonProps` with all these types?

File: packages/botonic-react/src/constants.js [hunk 3]
```diff
@@ -125,6 +125,7 @@ export const WEBCHAT = {
     // TriggerButton
     customTrigger: 'triggerButton.custom',
     triggerButtonImage: 'triggerButton.image',
+    triggerButtonNotifications: 'triggerButton.notifications',
```
Comment (asastre) [inline packages/botonic-react/src/constants.js:128]: Change: This should be changed to:
```suggestion
    triggerButtonNotificationsEnabled: 'triggerButton.notificationsEnabled',
```

**Review by asastre — CHANGES_REQUESTED** [review]

**Review by asastre — APPROVED** [review]

File: packages/botonic-react/src/components/index-types.ts (hunk 2 above)
Comment (Iru89) [reply packages/botonic-react/src/components/index-types.ts:154]: for now I do not apply it but I will take it into account later. Thanks Adria

**Pull request merged**